Description: Return MT_EXIT_FAILURE instead of MT_EXIT_INVOP for fatal exits from mt.

Index: cpio-2.11+dfsg/src/Makefile.am
===================================================================
--- cpio-2.11+dfsg.orig/src/Makefile.am
+++ cpio-2.11+dfsg/src/Makefile.am
@@ -38,6 +38,8 @@ cpio_SOURCES = \
  makepath.c\
  userspec.c
 
+mt_SOURCES = mt.c
+
 noinst_HEADERS =\
  cpio.h\
  cpiohdr.h\
Index: cpio-2.11+dfsg/src/mt.c
===================================================================
--- cpio-2.11+dfsg.orig/src/mt.c
+++ cpio-2.11+dfsg/src/mt.c
@@ -55,6 +55,8 @@
 
    David MacKenzie <djm@gnu.ai.mit.edu> */
 
+#include "configmake.h"
+
 #include <system.h>
 
 #include <stdio.h>
@@ -308,7 +310,7 @@ print_status (char *dev, int desc)
 void
 fatal_exit ()
 {
-  exit (MT_EXIT_INVOP);
+  exit (MT_EXIT_FAILURE);
 }
 
 int
